substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -316,17 +318,30 @@ impl<B: ChainApi, L: EventHandler<B>> Pool<B, L> {
 		let in_pool_hashes =
 			extrinsics.iter().map(|extrinsic| self.hash_of(extrinsic)).collect::<Vec<_>>();
 		let in_pool_tags = self.validated_pool.extrinsics_tags(&in_pool_hashes);
+		// Fill unknown tags based on the known tags from inactive views.
```

Comment:
Technically this module knows nothing about "view", let's make comment more generic:
```suggestion
		// Fill unknown tags based on the known tags given in `known_provides_tags`.
```